### ISSUE TITLE
If file is too large it shouldn't try to upload it

### ISSUE THREAD

**Issue by bjarnef** [ProfessorFarnsworth/HTML-5-File-Upload#15]
If I add a file larger the limit (4 MB), it is added to the list.
Could it maybe just ignore too large files, when you click "Begin Upload" or remove the files for you? And maybe disable the button if all files are too large for upload?
v. 0.2.1

![2015-02-08_16-10-57](https://cloud.githubusercontent.com/assets/2919859/6097041/769dcc86-afad-11e4-8625-024ed48e357b.jpg)

![2015-02-08_16-13-28](https://cloud.githubusercontent.com/assets/2919859/6097044/908014ce-afad-11e4-9948-22ee180389b1.jpg)

ohh, I then noticed that the upload icon have a higher z-index than the text it self.

![2015-02-08_16-15-28](https://cloud.githubusercontent.com/assets/2919859/6097050/fa9b9c2a-afad-11e4-9351-28580a93efbe.jpg)

![2015-02-08_16-15-51](https://cloud.githubusercontent.com/assets/2919859/6097051/032b3cce-afae-11e4-87e9-f99d94fc8739.jpg)

**Comment by ProfessorFarnsworth** [comment]
Yes of course! The files should not get queued. It might be a new thing due to my latest changes. Thanks for letting me know :)

**Comment by ProfessorFarnsworth** [comment]
Should be fixed :)

**Comment by bjarnef** [comment]
I see this issue in v. 0.3.1 too :)

![2015-02-11_19-04-56](https://cloud.githubusercontent.com/assets/2919859/6152933/2056bcd6-b221-11e4-9967-e57deee111ec.jpg)
![2015-02-11_19-05-33](https://cloud.githubusercontent.com/assets/2919859/6152934/205e98f2-b221-11e4-9cef-e96fa0a9f099.jpg)
